`make build-deps` and `make build-dev-deps` fails after PR #11

We are currently getting this error in CI. 
```
[ERROR] No switch is currently set. Please use 'opam switch' to set or install a switch
[ERROR] No repository tezos found
make: *** [Makefile:220: build-deps] Error 40
The command '/bin/sh -c make build-deps' returned a non-zero code: 2

Error: Process completed with exit code 2.
```
failed ci run: https://github.com/ocaml-multicore/tezos/runs/4069640771?check_suite_focus=true#step:3:1088 .

Locally, this manifests as 
```
[ERROR] Could not update repository "tezos": Git repository seems just initialized, try again after your
        first commit
```

--- a/scripts/version.sh
+++ b/scripts/version.sh
@@ -23,10 +23,11 @@ export full_opam_repository_tag=de2a372fc4d915bae85b7d2cc532a094941a9c0e
 
 ## opam_repository is an additional, tezos-specific opam repository.
 ## This value MUST be the same as `build_deps_image_version` in `.gitlab-ci.yml
-opam_repository_tag=0acf4a18dc69a98fd397d89626379d955691ffd5
-opam_repository_url=https://github.com/ocaml-multicore/tezos-opam-repository
-opam_repository_git=$opam_repository_url.git
-export opam_repository=$opam_repository_git\#opam_repository_tag
+export opam_repository_url=https://github.com/ocaml-multicore/tezos-opam-repository
+export opam_repository_tag=e1049e5477960f19c628d7e3808a6c75a658a9cf
+export opam_repository_git=$opam_repository_url.git
+export opam_repository=$opam_repository_git\#$opam_repository_tag
+
 
 ## for sapling param, fork is on github, url convention not the same in install_sapling_parameters
 raw_opam_repository_url=https://raw.githubusercontent.com/ocaml-multicore/tezos-opam-repository/4.12.0%2Bdomains

